### PR TITLE
Containerisation of System Monitoring Database

### LINKED ISSUE #4: Create Docker Container for System Monitoring Database
Issue raised by: developer

### What?
Create a Docker container to contain system performance database

### Where?
OpenStack cloud (temporary location) - 

### How?
* To ensure the database is modular and light weight in design 
* A test to see if containerising the Autoreduction database should be considered

### How to test the issue is resolved
* Ensure container correctly starts and ends
* Ensure database is accessible from container
* Ensure container is fully tested.

### PR DESCRIPTION
**Pull request by JackEAllen**
### Summary of work
Creation of Dockerfile, database schema image and documentation detailing containerisation of system monitoring database.

### Where
`database_containerisation/`

### How to test your work
* Ensure documented instructions are clear and concise.
* Manual Testing on local machine and/or SCD Cloud 

    For  manual testing on SCD Cloud, you will need to first follow the instructions detailed in [repository Wiki](https://github.com/ISISSoftwareServices/AutoreductionMonitor/wiki/Containerisation_setup) - Issue #5 
    * Ensure that the container created following README.md instructions, correctly mounts a data volume and contains tables from schema file.
    * Ensure that after adding new data, if you kill and remove the running database container, on re-creating the 
container again, data last added is persisted.

### Additional comments
* Please note that you will need Docker installed to test this PR
* As this is only for local use right now, credentials are visible in DockerFile. This will be templated using a credentials.ini file at a later point. 
* Travis will state the build has failed - This is simply because no tests have been added yet as the repository is very new.

Fixes #4 #5

### CODE REVIEW

**Comment by JackEAllen** [comment]
This is now ready for Review

File: .travis.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@ python:
 # command to install dependencies
 install:
   - pip install -r requirements.txt
-  
+
 # command to run tests
 script: pytest
```
Comment (ElliotAOram) [inline .travis.yml:12]: if there are no tests then just remove these for now (inc. coveralls statement)
Travis will then pass if (minimally) the project installs

File: database_containerisation/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM mysql:latest
```
Comment (ElliotAOram) [inline database_containerisation/Dockerfile:1]: Might be best to pin this to version to ensure the ground doesn't move from under our feet

**Comment by JackEAllen** [comment]
Not too sure how to get TravisCI to pass builds until tests are implemented

File: database_containerisation/system_monitoring.sql [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+--
+-- Table structure for table `instrument`
+--
+
+DROP TABLE IF EXISTS `instrument`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `instrument` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `instrument_name` varchar(255) DEFAULT NULL,
```
Comment (ElliotAOram) [inline database_containerisation/system_monitoring.sql:41]: this should just be `name` we already know it's an instrument by virtue of the table

**Pull request merged**